vimeo/vimeo-networking-java
# Quota error codes and content length check

#### Issue
The developer message wasn't getting set properly on the VimeoError object because the content length of the response was set to 0.
Additionally, we didn't have the error codes we needed in the enum.

#### Summary
I removed the content length check because it's not necessary. The following block is wrapped in a try catch so it shouldn't cause any issues (even in the worst case).
I also added in the error codes we needed.

#### How to Test
Make a POST request to `/me/videos` with the size parameter passed in (with a value that exceeds the weekly limit). See that you get a 4101 returned.
